process/smartContract/process.go

```diff
@@ -571,6 +586,7 @@ func (sc *scProcessor) ExecuteBuiltInFunction(
 		log.Debug("processed built in functions error", "error", err.Error())
 		return 0, err
 	}
+	builtInFuncGasUsed := vmInput.GasProvided - vmOutput.GasRemaining
```

Comment:
is it ok here not to use the "safe" function? 
I would use the same logic here as well even if it seems impossible, to keep it consistent.

Reply:
Done